website/docs/client.md

```diff
@@ -68,8 +68,8 @@ await accountsClient.getUser();
 
 ### Impersonating another user
 
-If you need to you can allow some users to impersonate the other users accounts eg: only admin users can impersonate to other users.
-Your current session will be replaced by another session.
+If you need to you can allow some users via [server authorization](/docs/server.md#authorize-to-impersonate) to impersonate the other users accounts, e.g.: Only admin users can impersonate to other users.
```

Comment:
Looks like the links are invalid and the website build is failing 🔴

Reply:
Hope this is fixed now.